test/app/accounts_test.rb

```diff
@@ -0,0 +1,46 @@
+require_relative '../app_helper'
+
+class AccountsTest < Minitest::Test
+  include Rack::Test::Methods
+  include AppTestHelper
+  include DBCleaner
+
+  def app
+    ExercismWeb::App
+  end
+
+  attr_reader :bob
+  def setup
+    super
+    @bob = User.create({
+      username: 'bob',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.